tests/plugins/pastefromword/manual/pasteimagessafari.md

```diff
@@ -0,0 +1,20 @@
+@bender-tags: bug, 4.16.0, 1134
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, basicstyles, pastefromword, elementspath, image, format, sourcearea
+
+1. Open [attached Word document](_assets/Image_safari.docx).
```

Comment:
It would be good to add a more complicated document with more image sources, especially gif and different flavors of jpg.

Reply:
How can I produce different flavors of JPG? :thinking:

Reply:
Actually we probably don't need to test different flavors of JPG, as all of its signatures starts with the same sequence: `FF D8 FF`. Checking if the image file starts with it should be enough for our case.